src/parameters.toml

```diff
@@ -491,6 +491,222 @@ alias = "k_thrshld_stpnss"
+alias = "Y_MTO3"
+value = 0.029
+type = "float"
+description = "Highly oxygenated organic molecule (HOM) percent yield from monoterpene (MT) oxidation with ozone, used for pure organic nucleation. This is an empirical fit to data. The highly oxygenated organic molecules concentration [HOM] is in 10^7 1/cm3, the monoterpene [MT], [OH], and [O3] concentrations are in TODO and the ion concentrations are in 1/cm3. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_Y_MTOH_percent]
+alias = "Y_MTOH"
+value = 0.012
+type = "float"
+description = "Highly oxygenated organic molecule (HOM) percent yield from monoterpene (MT) oxidation with hydroxyl radicals, used for pure organic nucleation. This is an empirical fit to data. The highly oxygenated organic molecules concentration [HOM] is in 10^7 1/cm3, the monoterpene [MT], [OH], and [O3] concentrations are in TODO and the ion concentrations are in 1/cm3. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_k_MTO3_organic_factor]
+alias = "k_MTO3"
+value =  8.05e-16
+type = "float"
+description = "Factor for temperature-dependent rate of monoterpene (MT) oxidation with ozone, used for pure organic nucleation. This is an empirical fit to data. The highly oxygenated organic molecules concentration [HOM] is in 10^7 1/cm3, the monoterpene [MT], [OH], and [O3] concentrations are in TODO and the ion concentrations are in 1/cm3. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_k_MTOH_organic_factor]
+alias = "k_MTOH"
+value =  1.2e-11
+type = "float"
+description = "Factor for temperature-dependent rate of monoterpene (MT) oxidation with hydroxyl radicals, used for pure organic nucleation. This is an empirical fit to data. The highly oxygenated organic molecules concentration [HOM] is in 10^7 1/cm3, the monoterpene [MT], [OH], and [O3] concentrations are in TODO and the ion concentrations are in 1/cm3. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_exp_MTO3_organic_factor]
+alias = "exp_MTO3"
+value = -640
+type = "float"
+description = "Additional factor for monoterpene (MT) oxidation with ozone, used for pure organic nucleation. This is an empirical fit to data. The highly oxygenated organic molecules concentration [HOM] is in 10^7 1/cm3, the monoterpene [MT], [OH], and [O3] concentrations are in TODO and the ion concentrations are in 1/cm3. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_exp_MTOH_organic_factor]
+alias = "exp_MTOH"
+value =  440
+type = "float"
+description = "Additional factor for monoterpene (MT) oxidation with hydroxyl radicals, used for pure organic nucleation. This is an empirical fit to data. The highly oxygenated organic molecules concentration [HOM] is in 10^7 1/cm3, the monoterpene [MT], [OH], and [O3] concentrations are in TODO and the ion concentrations are in 1/cm3.  From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_k_H2SO4org_factor]
```

Comment:
Is this a typo of `mam3_nucleation_k_H2SO4_organic_factor`?

Reply:
It's a parameter used for a different function. I can rename it so it's clearer.